services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -41,16 +46,40 @@ export const AppRoutes = () => {
           <Route path="*" element={<NotFoundPage />} />
           {/* MFP Report Routes */}
           {wpReport && (
-            <Route
-              path="/wp"
-              element={<DashboardPage reportType={ReportType.WP} />}
-            />
+            <>
+              <Route
+                path="/wp"
+                element={<DashboardPage reportType={ReportType.WP} />}
+              />
+              <Route
+                path="/wp/export"
+                element={
+                  userReportAccess["WP"] ? (
+                    <ExportedReportPage />
+                  ) : (
+                    <Navigate to="/" />
```

Comment:
I tried navigating straight to `/wp/export` as a state user and as an admin user, both within a report and outside. in all cases I was able to get there

Reply:
yeah that’s prob something we should patch up in both — if there’s not a currently selected report, redirect user back to dashboard. i’d say let’s make a new ticket for it — one for MFP, one for MCR.